h5p-bildetema/src/components/Language/Language.module.scss

```
@@ -0,0 +1,76 @@
+.language {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1.5rem;
+}
+
+.input_container {
+  border: 1px solid red;
+  position: relative;
+  padding-left: 1rem;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+.input_container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkmark {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+  box-shadow: 0 0 0 3px black;
+  border-radius: 5px;
+}
+
+.input_container input:checked ~ .checkmark {
+  background-color: black;
```

The checkbox will probably not be totally black, even though it's designed that way for now. Perhaps we should create a variable for this color.

```suggestion
  background-color: $checkbox-color;
```